testplan/testing/multitest/result.py

```diff
@@ -1404,27 +1404,47 @@ def log(self, message, description=None, category=None):
         :type message: ``str`` or instance
         :param description: Text description for the assertion.
         :type description: ``str``
-        :param category: Custom category for the assertion, set to 'email' to
-            include message in email exporter.
+        :return: ``True``
+        :rtype: ``bool``
+        """
+        entry = base.Log(message=message, description=description)
+        _bind_entry(entry, self)
+        return entry
+
+    def passing(self, description, category=None):
```

Comment:
I don't really see how this new `result.passing` is different from `result.log`? I don't think we need it, it's confusing.